frontend/src/screens/learner/card.tsx

```diff
@@ -126,150 +118,325 @@ const CompleteFlag: React.FC<StudyCardProps> = ({ study }) => {
+    backgroundColor: 'rgba(0, 0, 0, 0.2)',
+    width: '264px',
+    height: '350px',
+    [media.tablet]: {
+        width: '275px',
+        height: '360px',
+    },
+    [media.mobile]: {
+        width: '275px',
+        height: '360px',
+    }, 
+})
+
+const MultiSessionBack = styled(Box)<{ createshadow: boolean }> (({ createshadow }) => ({
+    position: 'absolute',
+    top: 8,
+    left: 8,
+    zIndex: '1',
+    backgroundColor: colors.white,
+    width: '264px',
+    height: '350px',
+    [media.tablet]: {
+        width: '275px',
+        height: '360px',
+    },[media.mobile]: {
+        width: '275px',
+        height: '360px',
+    },
+    transform: 'translateZ(-8px)',
+    border: `1px solid ${colors.lightGreen}`,
+    boxShadow: createshadow? '0px 4px 10px 0px rgba(0, 0, 0, 0.25)' : 'none',
+}))
+
+const StudyDetails: React.FC<{study: ParticipantStudy, hovered: Number, index: Number, position: { top: Number, left: Number }, setHovered: Function }> = ({ study, hovered, index, position, setHovered }) => {
+    
+    const api = useApi()
+    const env = useEnvironment()
+    const pi = getStudyPi(study)
+    const lead = getStudyLead(study)
+    const [isBusy, setIsBusy] = useState<boolean>(false)
+    const action = (study.stages?.length && !study.stages[0].completedAt) ? 'Start' : 'Continue'
 
-    if (!isMultiSession(study)) {
-        return <Space h='md' />
+    const onLaunch = async () => {
+        setIsBusy(true)
+        await launchStudy(api, study.id)
     }
 
     return (
-        <Box
-            className={cx({ 'xx-small': isMobile })}
-            justify='between'
-            wrap
-            margin={{ top: 'default' }}
-            css={{ minHeight: 35 }}
-        >
-            <MultiSession study={study} />
-        </Box>
+        <Flex pos='absolute' bg={colors.white} direction='column' align='flex-start' justify='flex-start' gap='lg'
+            display={hovered == index ? 'flex' : 'none'} w='548px' top={ position.top+'px' } left={ position.left+'px' } p='1.5rem'
+            style={{ zIndex: '5', boxShadow: '0px 4px 30px 0px rgba(0, 0, 0, 0.10)' }}>
+            <Flex display='flex' justify='space-between' w="100%">
+                <Text span fw='bolder' size='xl'>{study.titleForParticipants}</Text>
+                <Icon icon='close' color={colors.gray70} onClick={() => {
+                    setHovered(-1)
+                }}/>
+            </Flex>
+
+            <Text size='sm' c={colors.gray70}>{study.totalDuration} mins | {study.totalPoints} pts</Text>
+
+            <MultiSessionBar study={study} />
+
+            <StudyDetailsItem icon='info' title='About' desc={study.longDescription} />
+
+            <StudyDetailsItem icon='heart' title="What's in it for you?" desc={study.benefits} />
+
+            <StudyDetailsItem icon='feedback' title='Feedback' 
+                desc={`Feedback: ${study.stages?.length && study.stages[0].feedbackTypes?.reduce((acc, feedbackType, index) => {
```

Comment:
Lets try to move these things out of the template and into their own methods if possible

Reply:
Okay